dali/test/python/test_pipeline.py

```diff
@@ -1656,42 +1701,42 @@ def test_preserve_arg():
     pipe = dali.Pipeline(1, 1, 0)
     with pipe:
         out = dali.fn.external_source(source=[[np.array([-0.5, 1.25])]], preserve=True)
-        res = dali.fn.resize(out, preserve=True)
+        res = dali.fn.resize(out, preserve=True)  # noqa: F841
         pipe.set_outputs(out)
     pipe.build()
 
 
 def test_pipeline_wrong_device_id():
     pipe = dali.Pipeline(batch_size=1, num_threads=1, device_id=-123)
     with pipe:
-        pipe.set_outputs(np.int32([1,2,3]))
+        pipe.set_outputs(np.int32([1, 2, 3]))
     with assert_raises(RuntimeError, glob="wrong device_id"):
         pipe.build()
         pipe.run()
 
 
 def test_properties():
-    @dali.pipeline_def(batch_size=11, prefetch_queue_depth={"cpu_size":3, "gpu_size":2})
+    @dali.pipeline_def(batch_size=11, prefetch_queue_depth={"cpu_size": 3, "gpu_size": 2})
     def my_pipe():
         pipe = Pipeline.current()
         assert pipe.max_batch_size == 11
         assert pipe.batch_size == 11
         assert pipe.num_threads == 3
         assert pipe.device_id == 0
         assert pipe.seed == 1234
-        assert pipe.exec_pipelined == True
-        assert pipe.exec_async == True
-        assert pipe.set_affinity == True
+        assert pipe.exec_pipelined is True
+        assert pipe.exec_async is True
+        assert pipe.set_affinity is True
```

Comment:
```suggestion
        assert pipe.exec_pipelined is True
        assert pipe.exec_async is True
        assert pipe.set_affinity is True
```
```suggestion
        assert pipe.exec_pipelined
        assert pipe.exec_async
        assert pipe.set_affinity
```

Reply:
Again, I wanted this check to enforce the type of the attribute (to be boolean). I can't be sure that it was an initial intention of the author, but explicit comparison to True literal here made me think so. And it is an attribute of the object we define, so maybe we should test that its type does not unexpectedly change from boolean to something that is accidentally true-ish.